src/index.css

```diff
@@ -20,12 +20,17 @@
 }
 
 @font-face {
-  font-family: "IBMPlexSans";
+  font-family: "IBM Plex Sans Medium";
   font-style: normal;
   font-weight: 500;
-  font-display: swap;
-  src: url(https://fonts.gstatic.com/s/ibmplexsans/v19/zYX9KVElMYYaJe8bpLHnCwDKjSL9AIFsdA.woff2)
-    format("woff2");
+  src: url("./fonts/IBMPlexSans-Medium.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "IBM Plex Sans Regular";
```

Comment:
This should use the same font-family name, only the weight is different. https://stackoverflow.com/questions/28279989/multiple-font-weights-one-font-face-query

Reply:
Oh ok, but how do I identify them on `tailwind.config` the regular to the medium, I thought that it was with the font family name

Reply:
Font family + font weight will match the one you want.